docs/ruleset_functions/get_zone_peak_internal_load_floor_area.md

```diff
@@ -0,0 +1,48 @@
+# get_zone_peak_internal_load_floor_area
+**Schema Version:** 0.0.21
+**Description:** finds the peak coincident internal loads of a zone and returns the value in btu/h/ft2
+
+**Inputs:** 
+- **RMR**
+- **zone**
+
+**Returns:**  
+- **result**: an array giving 2 values: [total peak btu/h/ft2 in the zone, total zone area] the total peak btu/sf is the internal coincident peak loads in all spaces in the zone
+ 
+**Function Call:**
+
+
+## Logic:
+
+- create list of all loads in the zone.  Each load is an array of: [the peak in btu/h/ft2, schedule hourly values,schedule_multiplier], the peak in btu/h/ft2 takes into account the sensible and latent fraction, for where there are multiple internal_loads in a space, the schedule multiplier indicates the % of total of the internal load: `loads = []`
+- create an area variable which will be the sum of all space areas: `zone_area = 0`
+- For each space in zone: `for space in zone.spaces:`
+	- add the space.floor_area to the area variable: `zone_area += space.floor_area`
+	- create lights power variable for storing the total space lighting power: `lights_power = 0`
+	- For each lighting object in the space: `for lights in space.interior_lighting:`
+		- add the space lighting power to the lights_power variable & convert from W/m2 to btu/hr/ft2: `lights_power += lights.power_per_area * 0.316998331`
+	- create the array for the lights: `lights_info = [lights_power, space.normalize_interior_lighting_schedules]`
+	- append the lights_info to the loads list: `loads.append(lights_info)`
+	- get information for people & convert from W to btu/hr: `people_info = [space.number_of_occupants * space.occupant_sensible_heat_gain*3.412142, space.occupant_multiplier_schedule.hourly_values]`
+	- append people_info to the list: `loads.append(people_info)`
```

Comment:
@JasonGlazer we have occupant sensible heat gain but now latent heat gain. I don't remember if this was discussed by schema WG and is intentional.

Reply:
@JasonGlazer, seems like we need to add latent heat gains to occupants?

Reply:
I have created this issue- https://github.com/open229/ruleset-model-description-schema/issues/183 was unable to add Karen to this issue

Reply:
@mkarpman sorry, I missed this message. I get so many messages from GitHub, I sometimes miss the ones directed specifically to me.

Reply:
Please remove unit conversions per above.